lib/maplibre-compose/src/commonMain/kotlin/org/maplibre/compose/location/LocationPuck.kt

```diff
@@ -263,21 +272,27 @@ private fun rememberBearingAccuracyPainter(
 }
 
 @Composable
-private fun rememberLocationSource(locationState: UserLocationState): GeoJsonSource {
+private fun rememberLocationSource(
+  location: Location?,
+  bearing: BearingMeasurement?,
```

Comment:
```suggestion
  position: PositionMeasurement?,
  bearing: BearingMeasurement?,
  timestamp: TimeMark,
```

This one is private it should definitely should use what it needs, there is no "nicer public API" argument here